tests/lib/testing_wrapper.py

```diff
@@ -0,0 +1,102 @@
+import os
+import sys
+import six
+import json
+import pytest
+import tempfile
+import shutil
+
+from tests.lib.db_handler import DBHandler
+from tests.lib.file_handler import RemoteFileHandler
+
+
+class TestCase:
+
+    TEST_OPENPYPE_MONGO = "mongodb://localhost:27017"
+    TEST_DB_NAME = "test_db"
+    TEST_PROJECT_NAME = "test_project"
+    TEST_OPENPYPE_NAME = "test_openpype"
+
+    REPRESENTATION_ID = "60e578d0c987036c6a7b741d"
+
+    TEST_FILES = [
+        ("1eCwPljuJeOI8A3aisfOIBKKjcmIycTEt", "test_site_operations.zip", "")
+    ]
+
+    @pytest.fixture(scope='session')
+    def monkeypatch_session(self):
+        """Monkeypatch couldn't be used with module or session fixtures."""
+        from _pytest.monkeypatch import MonkeyPatch
+        m = MonkeyPatch()
+        yield m
+        m.undo()
+
+    @pytest.fixture(scope="module")
+    def download_test_data(self):
+        tmpdir = tempfile.mkdtemp()
+        for test_file in self.TEST_FILES:
+            file_id, file_name, md5 = test_file
+
+            f_name, ext = os.path.splitext(file_name)
+
+            RemoteFileHandler.download_file_from_google_drive(file_id,
+                                                              str(tmpdir),
+                                                              file_name)
+
+            if ext.lstrip('.') in RemoteFileHandler.IMPLEMENTED_ZIP_FORMATS:
+                RemoteFileHandler.unzip(os.path.join(tmpdir, file_name))
+
+            yield tmpdir
+            shutil.rmtree(tmpdir)
+
+    @pytest.fixture(scope="module")
+    def env_var(self, monkeypatch_session, download_test_data):
+        """Sets temporary env vars from json file."""
+        env_url = os.path.join(download_test_data, "input",
+                               "env_vars", "env_var.json")
+        if not os.path.exists(env_url):
+            raise ValueError("Env variable file {} doesn't exist".
+                             format(env_url))
+
+        env_dict = {}
+        try:
+            with open(env_url) as json_file:
+                env_dict = json.load(json_file)
+        except ValueError:
+            print("{} doesn't contain valid JSON")
+            six.reraise(*sys.exc_info())
+
+        for key, value in env_dict.items():
+            all_vars = globals()
+            all_vars.update(vars(TestCase))  # TODO check
+            value = value.format(**all_vars)
+            print("Setting {}:{}".format(key, value))
```

Comment:
Wondering if we need to ensure the `value` is string type ? Or force convert it `str(value)` ?
Also, isn't updating `globals()` dangerous ?